AutorestSwiftTest/ZZZAutoRestReportTest.swift

```diff
@@ -0,0 +1,158 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import AutoRestReport
+import AzureCore
+import Foundation
+import XCTest
+
+class ZZZAutoRestReportTest: XCTestCase {
+    var client: AutoRestReportClient!
+
+    // The 4 mobile test swerver swagger are:
+    // 1. custom-baseUrl.json
+    // 2. xms-error-response.json
+    // 3. body-integer.json
+    // 4. url.json
+    // The follow strings are the prefix of the mobile tests from TestServer report
+    let mobileTestsPrefix = [
+        // custom-baseUrl.json
+        "CustomBase",
+        // xms-error-response.json
+        "expected", "animalNotFoundError", "linkNotFoundError", "stringError", "intError",
+        // body-integer.json
+        "putInteger", "putLong", "putUnixTime", "getInteger", "getLong", "getUnixTime", "getInvalidUnixTime",
+        "getNullUnixTime",
+        // url.json
+        "UrlPathItem", "UrlQueries", "UrlPaths"
+    ]
```

Comment:
As we add new testserver swaggers, do we need to update this (I'm assuming yes).

Reply:
If the new test server swaggers are considered mobile scenarios, yes, we need to update this list. This list is gotten from the test server source code by looking at each endpoint and what is the name in the coverage report. that it built.